platform/security/certificate/security-ocsp-checker/src/test/java/org/codice/ddf/security/ocsp/checker/OcspCheckerTest.java

```diff
@@ -0,0 +1,291 @@
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.ocsp.checker;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.math.BigInteger;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.SecureRandom;
+import java.security.Security;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.util.Base64;
+import java.util.Calendar;
+import java.util.Date;
+import javax.security.auth.x500.X500Principal;
+import javax.ws.rs.core.Response;
+import org.apache.cxf.jaxrs.client.WebClient;
+import org.bouncycastle.asn1.x500.X500Name;
+import org.bouncycastle.asn1.x509.Certificate;
+import org.bouncycastle.asn1.x509.SubjectPublicKeyInfo;
+import org.bouncycastle.cert.X509CertificateHolder;
+import org.bouncycastle.cert.X509v3CertificateBuilder;
+import org.bouncycastle.cert.ocsp.BasicOCSPResp;
+import org.bouncycastle.cert.ocsp.CertificateStatus;
+import org.bouncycastle.cert.ocsp.OCSPException;
+import org.bouncycastle.cert.ocsp.OCSPReq;
+import org.bouncycastle.cert.ocsp.OCSPResp;
+import org.bouncycastle.cert.ocsp.RevokedStatus;
+import org.bouncycastle.cert.ocsp.SingleResp;
+import org.bouncycastle.cert.ocsp.UnknownStatus;
+import org.bouncycastle.operator.ContentSigner;
+import org.bouncycastle.operator.jcajce.JcaContentSignerBuilder;
+import org.codice.ddf.cxf.client.ClientFactoryFactory;
+import org.codice.ddf.cxf.client.SecureCxfClientFactory;
+import org.junit.Assert;
+import org.junit.Test;
+import org.osgi.service.event.EventAdmin;
+
+public class OcspCheckerTest {
+
+  private final ClientFactoryFactory factory = mock(ClientFactoryFactory.class);
+  private final EventAdmin eventAdmin = mock(EventAdmin.class);
+
+  @Test
+  public void testConvertingX509CertificatesToBCCertificates() throws Exception {
+    OcspChecker ocspChecker = new OcspChecker(factory, eventAdmin);
+    X509Certificate x509Certificate = getX509Certificate();
+
+    Certificate certificate = ocspChecker.convertToBouncyCastleCert(x509Certificate);
+    assertNotNull(certificate);
+    Assert.assertEquals(
```

Comment:
✏️ I wouldn't block the PR on this, but the best practice is to avoid using JUnit's assert methods and instead use hamcrest's `assertThat` methods. This applies to this whole class, but this line, to use as an example, would look something like
```assertThat(x509Certificate.getSerialNumber(), equalTo(certificate.getSerialNumber().getValue()));```

See https://codice.atlassian.net/wiki/spaces/DDF/pages/6291523/Unit+Testing#UnitTesting-Assertions